api/parsers.go

```diff
@@ -181,6 +249,74 @@ func parseExitFilters(c querier) (*common.TokenID, *ethCommon.Address, *babyjub.
 	return tokenID, addr, bjj, idx, nil
 }
 
+func parseTxsFilters(c querier) (*common.TokenID,
```

Comment:
Please, could you merge all the return arguments apart from the error (`*common.TokenID,
	*ethCommon.Address, *ethCommon.Address, *ethCommon.Address,
	*babyjub.PublicKeyComp, *babyjub.PublicKeyComp, *babyjub.PublicKeyComp,
	*common.Idx, *common.Idx, *common.Idx`) into a single struct?  And then return just `(*StructName, error)`
	I think it will make the code much clearer